modules/install-sno-ibm-power.adoc

```diff
@@ -0,0 +1,167 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
```

Comment:
Can I suggest you take a look at my attempt to re-organize this module to make it more OpenShift? 

https://github.com/openshift/openshift-docs/pull/66443/files#diff-aa23555a845411d004c83340cdbdf8bdf7b64ef6b6642ceed1aacc03c648415c

Maybe you can ask someone else on OpenShift docs the team to take a look? This is a tricky Prerequisite ad Procedure.